dll/cpl/sysdm/lang/ja-JP.rc

```diff
@@ -19,7 +19,7 @@ BEGIN
     LTEXT "", IDC_MACHINELINE4, 130, 155, 118, 9
     LTEXT "", IDC_MACHINELINE5, 130, 164, 118, 9
     LTEXT "", IDC_MACHINELINE6, 130, 173, 118, 9
-    CONTROL "<A HREF=""http://www.reactos.org/"">ReactOS ホームページ</A>へ行く", IDC_ROSHOMEPAGE_LINK, "SysLink", WS_CHILD | WS_VISIBLE | WS_TABSTOP, 10, 200, 140, 10
+    CONTROL "<A HREF=""https://reactos.org"">ReactOS ホームページ</A>へ行く", IDC_ROSHOMEPAGE_LINK, "SysLink", WS_CHILD | WS_VISIBLE | WS_TABSTOP, 10, 200, 150, 10
```

Comment:
Try to keep the same URL as in the other translations: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDC_ROSHOMEPAGE_LINK

Reply:
Reverted. OK.